charts/opennotificaties/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: opennotificaties
 description: API voor het routeren van notificaties
 
 type: application
-version: 1.8.2
+version: 1.8.3
 appVersion: 1.8.0
```

Comment:
I see that the change in https://github.com/open-zaak/open-notificaties/pull/266/commits/645ef9b799f6bd7922e11a2441f4c8d89f8e5a0a is only present in main.

For this to be usable by the Helm chart, the appVersion should be bumped to a version that contains that change.